runtime/calamari/src/lib.rs

```diff
@@ -136,26 +136,21 @@ parameter_types! {
 pub struct BaseFilter;
 impl Filter<Call> for BaseFilter {
 	fn filter(c: &Call) -> bool {
-		!matches!(
-			c,
-			// Monetary
-			Call::Assets(pallet_assets::Call::create(..)) | Call::Balances(_) |
-			// Filter these calls to prevent users from creating assets or making transfers.
-			// Core
-			Call::System(_) |
-			// Filter these calls to prevent users from runtime upgrade without sudo privilege.
-			// pallet-timestamp(_) and parachainSystem(_) could not be filtered because they are used in commuication between releychain and parachain.
-			// Utility
-			Call::Utility(_) | Call::Multisig(_) |
-			// Filter these calls to prevent users from utility operation.
-			Call::Authorship(_) |
+		match c {
+			Call::Timestamp(_) => true,
```

Comment:
Can you merge lines of code from line140 to 145 like this:
```rust
Call::Timestamp(_) | Call::ParachainSystem(_) | Call::Authorship(_) | Call::Sudo(_) => true,
```

Reply:
Sure, already change

Reply:
No need to use `{}`, directly return true.

Reply:
It's automatically change by cargo fmt

Reply:
Fine.